### DISCUSSION TITLE
docs/go/a1-style-guide/

### DISCUSSION THREAD

**Discussion by giscus** [huanlin/huanlin.github.io#46] [Announcements]
# docs/go/a1-style-guide/

Note
這篇筆記還沒整理完，先別看喔。
Package name 好的套件名稱應簡潔明白，通常是單數名詞，而且全都是用英文小寫。注意不可以用底線（snake case）或大小寫混和（mixedCaps）。

https://huanlin.cc/docs/go/a1-style-guide/

**Comment by huanlin** [comment]
https://google.github.io/styleguide/go/index
